### PR TITLE
Update snowcrash

### PR DESCRIPTION
**Pull request by klokane**
- update snowcrash to MSON branch
- add modules cmdline && sos
- fix warning on gcc 4.6 in tests

### CODE REVIEW

File: drafter.gyp [hunk 1]
```diff
@@ -29,18 +42,17 @@
 
       # FIXME: replace by direct dependecies
```
Comment (pksunkara) [inline drafter.gyp]: This warning should be removed, right?

File: drafter.gyp [hunk 2]
```diff
@@ -86,19 +98,19 @@
...
```
Comment (pksunkara) [inline drafter.gyp]: Similarly, this warning.

**Comment by pksunkara** [comment]
Just 2 minor things I want you to address before I can merge.

**Comment by klokane** [comment]
Still in backlog. Will fix in future

**Comment by pksunkara** [comment]
> Still in backlog.

Okay, but could you explain what exactly they are about? I think I misunderstood what they are about.

**Comment by klokane** [comment]
See chapter *direct_dependent_settings* 

https://code.google.com/p/gyp/wiki/InputFormatReference#Dependencies_and_Dependents

In short it allow "inject" dependencies like include dirs, define and other to other targets. 
eg. there should be direct dependencies in `libdrafter` target and `drafter` shoud use them instead of repeat include paths. But while there is `common.gypi` included, direct dependencies doesn't works. 

Still have no time to investigate for reason.

**Pull request merged**